modules/cron4s/src/test/scala/pureconfig/module/cron4s/Cron4sSuite.scala

```diff
@@ -24,7 +24,7 @@ class Cron4sSuite extends BaseSuite {
 
     val errors = ConfigReaderFailures(
       ConvertFailure(
-        CannotConvert("10-65 * * * * *", "CronExpr", "Expected '\" \" | 0-59' at position 2 but found '\"-65 * * * * *\"'"),
+        CannotConvert("10-65 * * * * *", "CronExpr", "blank expected at position 3 but found '-'"),
```

Comment:
Unfortunately this will fail for Scala 2.11, which still uses the old version :( You can choose to either not check the error message or drop Scala 2.11 support.

Reply:
what about dropping 2.11 support? not sure if there are any big changes in the core of `pureconfig` but in regards to cron4s, it could be argued that users could use the previous module version if they need to run with 2.11...

Reply:
Yes, I tend to agree. While we'll support Scala 2.11 in core for some more time, I think it wouldn't be critical for `pureconfig-cron4s` to stop being updated.